DOL.WHD.Section14c.Web/src/modules/components/attachmentField/attachmentFieldTemplate.html

```diff
@@ -1,8 +1,10 @@
 <div ng-show="attachmentId == undefined">
-  <input id="{{::$id}}_FileUpload" class="upload-file-input" type="file" ng-hide="true" onchange="angular.element(this).scope().vm.onAttachmentSelected(this)" accept=".doc,.docx,.xls,.xlsx,.pdf,.jpg,.jpeg,.png">
-  <label for="{{::$id}}_FileUpload" class="upload-label usa-button-primary">Browse</label>
-  <input id="{{::$id}}" name="{{::$id}}" class="upload-input" type="text" disabled>
-  <label class="underlabel" for="{{::$id}}">File types accepted: PDF, Word, Excel, JPG, PNG</label>
+  <div class="dol-file-upload-ui">
+    <div aria-hidden="true" class="usa-button-primary">Browse</div>
+    <input class="dol-file-upload-input" id="{{::$id}}_FileUpload" class="upload-file-input" type="file" onchange="angular.element(this).scope().vm.onAttachmentSelected(this)" accept=".doc,.docx,.xls,.xlsx,.pdf,.jpg,.jpeg,.png">
+    <div class="dol-file-upload-focus"></div>
```

Comment:
I can't figure out how clicking that `div` button triggers the file browser to pop up, but it does and that's the important thing. 😛

Reply:
The `div` doesn't trigger the file browser popup. :smile: The user is actually clicking or keyboard accessing the file upload input field. Visual users won't know it because all they see is the div that looks like a button.

Reply:
Ooooh, clever!  I like it.  😄